mitodl/bootcamp-ecommerce
# Add application submissions admin api

#### What are the relevant tickets?
Closes #462 

#### What's this PR do?
Adds an API to view bootcamp submissions:
- Submissions can be filtered by review status and/or the bootcamp
- The response can be sorted by the created_data (asc or desc) of the bootcamp submission

#### How should this be manually tested?
- Go to `/api/submissions/`, you should see existing application submission steps listed there.
- You'll see a "Filters" button in the top right of the browseable API UI that will let you test out the filtering and sorting capabilities added here.
- You should see "facet" data in the response that updates as you filter results..
- The detail `/api/submissions/:id/` API should return a single result
